src/main/java/de/torui/coflsky/EventHandler.java

```diff
@@ -188,4 +202,165 @@ private static void ProcessScoreboard(String line){
         }
     }
 
+    private static class InventoryWrapper {
+        public String chestName;
+        public String fullInventoryNbt;
+    }
+    private class DescModification {
+        public String type;
+        public String target;
+        public String value;
+    }
+
+    public static HashMap<ItemStack, DescModification[]> tooltipItemMap = new HashMap<ItemStack, DescModification[]>();
+    public static HashMap<String, DescModification[]> tooltipItemUuidMap = new HashMap<String, DescModification[]>();
+    public static HashMap<String, DescModification[]> tooltipItemIdMap = new HashMap<String, DescModification[]>();
+
+    public static final DescModification[] EMPTY_ARRAY = new DescModification[0];
+    public static final NBTTagCompound EMPTY_COMPOUND = new NBTTagCompound();
+
+    public static String ExtractStackableIdFromItemStack(ItemStack stack) {
+        if (stack != null) {
+            try {
+                String uuid = stack.serializeNBT().getCompoundTag("tag").getCompoundTag("ExtraAttributes")
+                        .getString("id");
```

Comment:
make the return value include the item count as the description can differ based on the amount in a stack